app/utils/validators/forms/FieldValidator.scala

```diff
@@ -136,4 +136,9 @@ object FieldValidator {
 
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
+
+  val REGEX = "^(GB|gb)[\\d]+$"
```

Comment:
Please, change this name to be more descriptive.

Reply:
fixed as a similar review comment above.

Reply:
Made inline...removed val